components/Thanks.tsx

```diff
@@ -7,7 +7,14 @@ export const Thanks: React.FC<{ close: Function }> = ({ close }) => (
     <h2 className="mt-4 mb-4 pt-3 pb-3 fw-bold text-center">
       Thanks for letting us know!
     </h2>
-    <Button type="primary" color="white" size="lg" onClick={close}>
+    <Button
+      btnType="primary"
+      color="white"
+      size="lg"
+      onClick={() => {
```

Comment:
Why is there an update from `onClick={close}` to 

```
onClick={() => {
  close()
 }}
```
?

Is there an error otherwise?

Reply:
> Why is there an update from `onClick={close}` to
> 
> ```
> onClick={() => {
>   close()
>  }}
> ```
> 
> ?

With `onclick={close}` I was getting this error:
`Type 'Function' is not assignable to type 'MouseEventHandler<HTMLButtonElement>'.
  Type 'Function' provides no match for the signature '(event: MouseEvent<HTMLButtonElement, MouseEvent>): void'.`

Changing it seemed to fix the issue.